book/A-git-in-other-environments/sections/guis.asc

```diff
@@ -1,112 +1,109 @@
 
-It's important to note that different interfaces are tailored for different workflows.
-Some clients expose only a carefully curated subset of Git functionality, in order to support a specific way of working that the author considers effective.
-When viewed in this light, none of these tools can be called ``better'' than any of the others, they're simply more fit for their intended purpose.
-Also note that there's nothing these graphical clients can do that the command-line client can't; the command-line is still where you'll have the most power and control when working with your repositories.
+Mahalagang tandaan na ang iba't ibang mga interface ay natatangi para sa iba't-ibang mga workflow.
+Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
+Kapag tiningnan sa liwanag na ito, wala sa mga tool na ito ay maaaring masabi na ``mas mabuti'' kaysa sa iba pa, ang mga ito ay mas angkop sa kanilang layunin.
+Tandaan din na wala sa mga graphical na kliyente na maaaring gawin na ang command-line client ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
 
-==== `gitk` and `git-gui`
+==== `gitk` at `git-gui`
 
 (((git commands, gitk)))(((git commands, gui)))(((gitk)))
-When you install Git, you also get its visual tools, `gitk` and `git-gui`.
+Kapag nag-install ka ng Git, nakakuha ka rin ng mga visual na tool nito, `gitk` at `git-gui`.
 
-`gitk` is a graphical history viewer.
-Think of it like a powerful GUI shell over `git log` and `git grep`.
-This is the tool to use when you're trying to find something that happened in the past, or visualize your project's history.
+`gitk` ay isang graphical viewer ng kasaysayan.
+Isipin ito tulad ng isang malakas na shell ng GUI sa `git log` at `git grep`.
+Ito ang tool na gagamitin kapag sinusubukan mong maghanap ng isang bagay na nangyari sa nakaraan, o maisalarawan ang kasaysayan ng iyong proyekto.
 
-Gitk is easiest to invoke from the command-line.
-Just `cd` into a Git repository, and type:
+Ang Gitk ay pinakamadaling tawagin mula sa command-line.
+`cd` lamang sa isang repository ng Git, at i-type:
 
 [source,console]
 ----
 $ gitk [git log options]
 ----
 
-Gitk accepts many command-line options, most of which are passed through to the underlying `git log` action.
-Probably one of the most useful is the `--all` flag, which tells gitk to show commits reachable from _any_ ref, not just HEAD.
-Gitk's interface looks like this:
+Tinatanggap ng Gitk ang maraming mga opsyon sa command line, na karamihan ay naipasa sa pinagbabatayan na aksyon ng `git log`.
+Marahil ang isa sa mga pinaka-kapaki-pakinabang ay ang `--all` na flag, na nagsasabi sa gitk na ipakita ang mga commit na mula sa _any_ ref, hindi lamang HEAD.
```

Comment:
`_anumang_`

Reply:
I didnt changed it to _anumang_ because is a _any_ is a reserved word.